src/theme.tsx

```diff
@@ -168,37 +174,129 @@ export default mergeTheme(defaultTheme, {
         backgroundColor: hoverIndicator,
       },
     },
-    ({ body0 }: any) => body0 && {
-      fontSize: 18,
-      lineHeight: '28px',
+    ({ h1 }: any) => h1 && {
+      fontFamily: fontFamilies.semi,
+      fontSize: 72,
+      lineHeight: '110%',
+      fontWeight: 400,
+      letterSpacing: '-1px',
```

Comment:
Are you certain you don't wanna use the H1 component?

Reply:
I think it's good to decouple the styles from the semantic meaning of specific html tags like <h1>,<h2>, etc. Or is this an instance where you could do something like <H1 as={H3}> ?